### PR TITLE
Add clarifications after reading Nicolas' JEP draft

### PR DESCRIPTION
**Pull request by daniel-beck**
Motivated by https://github.com/jenkinsci/jep/pull/31

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Contributor is probably fine. But a "Sponsor" needs to be a human being for sure.

File: jep/1/README.adoc [hunk 1]
```diff
@@ -152,6 +152,7 @@ A JEP may have any number of "Contributors" who
 help write, implement, discuss, or offer feedback about that JEP.
 One contributor might do only one or any combination of these
 of these things during any part of the life of a JEP.
+Companies or legal entitites cannot be contributors.
```
Comment (rtyler) [inline jep/1/README.adoc]: I think I mirror @oleg-nenashev's concerns here. This is off-putting for corporate contribution IMHO.

File: jep/1/README.adoc [hunk 2]
```diff
@@ -189,6 +190,7 @@ The BDFL Delegate for a JEP has all the responsibilities of the BDFL within the
 except that BDFL Delegate may not delegate to someone else - there is no such thing as a "BDFL Delegate Delegate".
 
 A BDFL Delegate may be selected at any point before the JEP is reviewed.
+A record of this selection must be available publicly.
```
Comment (rtyler) [inline jep/1/README.adoc]: This looks good :+1: \

File: jep/1/README.adoc (hunk 1 above)
Comment (daniel-beck) [reply jep/1/README.adoc]: This was the most straightforward way to prevent something "Sponsor: Praqma" in #31. I'll probably not email info@praqma.com, or ping @Praqma, about concerns about a JEP.

Would it help to rephrase this to something like the following?

> While we invite contributions by companies or other organizations, contributors (such as sponsors or BDFL Delegates below) need to be specific people.

Comment (rtyler) [reply jep/1/README.adoc]: I think that's much better, yes.

(Soylent) Jenkins is made of people!

Comment (oleg-nenashev) [reply jep/1/README.adoc]: @daniel-beck Works for me, 👍 
We could also allow mentioning affiliation directly in JEP. E.g. `[Oleg Nenashev](GitHub link), [CloudBees, Inc.](Website link)` in the doc

**Comment by daniel-beck** [comment]
Addressed @oleg-nenashev and @rtyler review comments.
